docs/cli/stop.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "wash stop"
+draft: false
+sidebar_position: 24
+description: "wash stop command reference"
+--- 
+
+`wash stop` assists in stopping an actor or a provider from an OCI reference on a host. `wash stop` also helps in stopping hosts. The user may specify if they want to stop an actor or a provider and pass the OCI reference of that entity to the command. To specify a host to launch the actor or provider, pass the host ID using the `--host-id` flag. Following are the available subcommads:
```

Comment:
Consider combining the first two sentences: "`wash stop` allows you to stop actors, providers, and hosts"

Also: s/launch/stop